sigstore/verify/models.py

```diff
@@ -123,24 +135,31 @@ class VerificationMaterials:
     certificate: Certificate
     """
     The certificate that attests to and contains the public signing key.
+
+    # TODO: Support a certificate chain here, with optional intermediates.
     """
 
     signature: bytes
     """
     The raw signature.
     """
 
-    _offline_rekor_entry: LogEntry | None
+    offline: bool
+    """
+    Whether to do offline Rekor entry verification.
+
+    This is a slightly weaker verification verification mode, as it demonstrates
+    that an entry has been signed by the log but not necessarily included in it.
     """
```

Comment:
Flagging: I'm not super happy with this setting being here, since it's conceptually a verification option rather than a material. 

The other possibility I can think of is a separate `VerificationOptions` model, but that might be overkill given that this is currently the only option we support. But maybe there will be more in the future?

Reply:
Personally, I think it's ok.

If we go for a separate `VerificationOptions` model, we're never going to collapse that back into `VerificationMaterials` even if it remains as one option for a very long time because there's always going to be the possibility of more.

Reply:
Yeah, that's a good point. I think what I'll do it make it "private" like `_rekor_entry` so that it isn't part of our public API commitment. It'll still end up being stabilized as part of the methods on `VerificationMaterials`, but that'll at least avoid the need to change this model's public fields if we ever do decide to refactor it.